packages/amplify_datastore_plugin_interface/lib/src/types/models/model.dart

```diff
@@ -17,18 +17,14 @@ import 'model_schema.dart';
 import 'model_schema_definition.dart';
 
 abstract class Model {
-  final String id;
-
   ModelType getInstanceType();
 
-  const Model({required this.id});
-
-  String getId() {
-    return id;
-  }
+  String getId();

```

Comment:
question: Why continue to include this method written this way? Extended classes will continue to use @override, correct? Not requesting a change, just trying to understand.

Reply:
We need to include this method since Model is an abstract class.  And abstract class needs to have method so we can call it without need to cast to the underlying Model type like Blog, Post, etc.

Reply:
thanks